tools/eval_metric.py

```diff
@@ -146,9 +161,9 @@ def eval_metric(args):
 
 
 @torch.no_grad()
-def evaluate(model, data_loader, coco_evaluator, device, print_freq):
+def evaluate(model, data_loader, coco_evaluator, device, print_freq, use_wandb, wandb_project, wandb_entity):
     # COCO evaluation
-    metric_logger = MetricLogger(delimiter="  ")
+    metric_logger = MetricLogger(args, delimiter="  ")
```

Comment:
Hi @SauravMaheshkar , Seems this `args` are forgotten to be adapted?

Reply:
Sorry missed that. Fixed in [a7a46f0](https://github.com/zhiqwang/yolov5-rt-stack/pull/426/commits/a7a46f06184684b2bcddbfc6c5cc9396007e74fa)